pkg/markets/models.go

```diff
@@ -2,23 +2,21 @@ package markets
 
 import "time"
 
-type Location struct {
-	City    string `json:"city,omitempty"`
-	Country string `json:"country,omitempty"`
-	State   string `json:"state,omitempty"`
-}
-
 type Market struct {
-	ID                        string                           `json:"marketID"`
+	ID                        string                           `json:"marketID" bson:"marketID"`
```

Comment:
This doesn't seem to have changed at all other than location now being embedded into the struct directly, and the removal of the Location struct here could actually break dependent systems (anything that actually creates an instance of a Location for the purposes of building a Market to send to the market-api (I don't think this actually is happening anywhere, but wanted to note that this isn't a change with zero potential side effects).

The `bson` tags on the struct are also added, but those aren't needed in the SDK (we don't serialize to or from BSON from these structs). Is there a reason for this change?